### PR TITLE
[11.0] [MIG] Account Invoice Refund Option

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**
Migration of account_invoice_refund_option from 9.0 to 11.0
#317

### CODE REVIEW

**Review by acsonefho — APPROVED** [review]
LGTM about code

File: account_invoice_refund_option/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Create Refund Invoice",
+    "summary": "Allows you to create directly a refund without starting from "
+               "an invoice",
+    "version": "11.0.1.0.0",
+    "category": "Accounting",
+    "website": "https://www.tecnativa.com/",
```
Comment (nikul-serpentcs) [inline account_invoice_refund_option/__manifest__.py:9]: @ThomasBinsfeld Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

File: account_invoice_refund_option/tests/test_is_refund.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+from odoo.tests.common import SavepointCase
+
+
+class IsRefundCase(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        """Add some defaults to let the test run without an accounts chart."""
+        super(IsRefundCase, cls).setUpClass()
+        cls.AccountInvoice = cls.env["account.invoice"]
+        cls.journal_sale = cls.env["account.journal"].create({
```
Comment (nikul-serpentcs) [inline account_invoice_refund_option/tests/test_is_refund.py:13]: add the centrally variable and use Object

**Pull request closed without merging**